shared/src/hooks/useFetchSolVaultData.tsx

```diff
@@ -28,12 +32,21 @@ const useFetchSolVaultData = (): SolanaVaultData => {
       } = await vaultData.getVaultData(connection, vaultAddress);
 
       let lockedBalanceInAsset = BigNumber.from(0);
+      let depositBalanceInAsset = BigNumber.from(0);
+      let totalQueuedWithdrawals = BigNumber.from(0);
       const totalQueuedDeposits = BigNumber.from(vault.totalQueuedDeposits);
-      const totalQueuedWithdrawals = BigNumber.from(
-        vault.totalQueuedWithdrawals
-      );
 
```

Comment:
Just a comment here : 

This is using the cached state in Vault.depositQueue, so the latest state is only after you call `Vault.updateDepositAndWithdrawalQueues()` but I don't imagine it would change very often.